src/containers/BookList.js

```diff
@@ -1,38 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-
-const BooksList = ({ books }) => (
-
-  <table>
-
-    <thead>
-      <tr>
-        <th>Book Id</th>
-        <th>Title</th>
-        <th>Category</th>
-      </tr>
-    </thead>
-    <tbody>
-      {books.bookReducer.map((book) => (
-
-        <tr key={book.id + 2}>
-          <td key={book.id}>{book.id}</td>
-          <td key={book.title}>{book.title}</td>
-          <td key={book.category}>{book.category}</td>
+import { deleteBook } from '../actions';
+import Book from '../components/Book';
+
+const BooksList = ({ books, deleteBook }) => {
+  const handleRemoveBook = (book) => {
+    console.log(book);
```

Comment:
## Optional
Kindly remove the console.log() statement!!! It is not good for production